tools/test/audio/process_test.m

```diff
@@ -139,18 +139,15 @@
 	fprintf('Number of skipped tests = %d\n', r.n_skipped);
 	fprintf('============================================================\n');
 
-	n_fail = r.n_fail;
-	n_pass = r.n_pass;
-	n_na = r.n_na;
-	if r.n_fail < 1 && r.n_pass < 1
-		fprintf('\nERROR: No test results.\n');
```

Comment:
I'd like to keep the error message about no tests executed. What problem did this cause?

Reply:
I don't believe there is a problem . `Test not run` should be the interpretation. 
I am OK, I'II include back No test results case.